crates/holochain_p2p/src/test.rs

```diff
@@ -218,573 +206,3 @@ fixturator!(
         HolochainP2pDnaFixturator::new(Empty).next().unwrap()
     };
 );
-
-#[cfg(test)]
```

Comment:
Equivalent tests elsewhere or do these need replacing? Or maybe just not considered useful because we test this enough through Holochain tests?

Reply:
The important parts of these tests have all been replaced by the integration tests in `crates/holochain_p2p/tests/tests`. The unimportant parts of these tests were manufacturing data in the test itself and then verifying that data on the other side.